packages/contracts/test/LibCycleTurns.t.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { MudV2Test } from "@latticexyz/std-contracts/src/test/MudV2Test.t.sol";
+import { LibCycleTurns } from "../src/cycle/LibCycleTurns.sol";
+import { CycleTurns, CycleTurnsLastClaimed, ActiveGuise, ActiveCycle } from "../src/codegen/Tables.sol";
+
+contract LibExperienceTest is MudV2Test {
+  bytes32 internal targetEntity = keccak256("targetEntity");
+
+  uint32 cycleEntity;
+  uint32 initialTurns;
+  bytes32 guiseProtoEntity;
+
+  function setUp() public virtual override {
+    super.setUp();
+    // TODO remove this if it's integrated into MUD
+    vm.startPrank(worldAddress);
+    cycleEntity = LibCycleTurns.initActiveCycle(targetEntity);
```

Comment:
just use keccak256 like with targetEntity

Reply:
perhaps I misunderstood you, the tests are run, but they do not pass, because `keccak256` is initialized incorrectly, for example, the values 0 and 10 are compared in the `test_setUp` function

Reply:
cycleEntity is fine now